sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -574,11 +583,15 @@ export function fromRheaMessage(
       -readonly [P in keyof T]: T[P];
     }
   >;
-  const props: PartialWritable<ServiceBusReceivedMessage> = {};
+  const props: PartialWritable<ServiceBusReceivedMessage> & { state: "active" | "deferred" | "scheduled" } = { state: "active" };
   if (rheaMessage.message_annotations != null) {
     if (rheaMessage.message_annotations[Constants.deadLetterSource] != null) {
       props.deadLetterSource = rheaMessage.message_annotations[Constants.deadLetterSource];
     }
+    const messageState = rheaMessage.message_annotations[Constants.messageState];
+    if (isDefined(messageState) && (messageState === 1 || messageState === 2)) {
```

Comment:
Do we need the `isDefined` check here? Won't the below just work without the defined check?
```js
if (messageState === 1) {
   props.state = "deferred";
} else if (messageState === 2) {
   props.state = "scheduled";
}
```

Reply:
Yeah this is enough. IsDefined() is from previous iteration that no longer needed.